python/pyspark/sql/functions.py

```diff
@@ -7619,6 +7619,36 @@ def get(col: "ColumnOrName", index: Union["ColumnOrName", int]) -> Column:
     return _invoke_function_over_columns("get", col, index)
 
 
+@try_remote_functions
+def array_prepend(col: "ColumnOrName", element: Any) -> Column:
```

Comment:
```suggestion
def array_prepend(col: "ColumnOrName", value: Any) -> Column:
```

to be consistent with `array_append`